staging/src/k8s.io/code-generator/cmd/validation-gen/output_tests/forbidden/doc.go

```diff
@@ -32,13 +32,11 @@ type T1 struct {
 	// +forbidden
 	PS *string `json:"ps"`
 
-	// +forbidden
```

Comment:
shouldn't we keep this to prove that it DOESN'T emit code?

Reply:
it would result in code-gen failure because we error it out. I feel it is similar to the test of targeting "compile error"?

Reply:
oh, of course, sorry.  I was thinking about required but looking at forbidden.